File: src/main/java/um/haberes/report/model/haberes/core/DocenteDesignacionDto.java
package um.haberes.report.model.haberes.core;

import java.math.BigDecimal;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class DocenteDesignacionDto {

    private Long legajoId;

    private Integer anho;

    private Integer mes;

    private Integer facultadId;

    private Integer geograficaId;

    private String espacio;

    private BigDecimal horasSemanales;

    private String cargo;

    private String designacion;

    private BigDecimal horasDesignacion;

    private Byte anual;

    private Byte semestre1;

    private Byte semestre2;

}

File: src/main/java/um/haberes/report/kotlin/dto/haberes/core/DocenteDesignacionDto.kt
package um.haberes.report.kotlin.dto.haberes.core

import java.math.BigDecimal

data class DocenteDesignacionDto
    (
    var legajoId: Long? = null,
    var anho: Int? = null,
    var mes: Int? = null,
    var facultadId: Int? = null,
    var geograficaId: Int? = null,
    var espacio: String? = null,
    var horasSemanales: BigDecimal? = null,
    var cargo: String? = null,
    var designacion: String? = null,
    var horasDesignacion: BigDecimal? = null,
    var anual: Byte? = null,
    var semestre1: Byte? = null,
    var semestre2: Byte? = null,
            ){
}